### PR TITLE
Added to README, Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hoo761**
Added to this PR:
- Sections to README including team section
- Contributing document
- License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
+**[Installation](#installation)** |
```
Comment (saulshanabrook) [inline README.md:1]: This looks great!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Contributing to JupyterLab
```
Comment (saulshanabrook) [inline CONTRIBUTING.md:1]: There are a few tools in here that we don't have set up yet in this repo (testing, prettier, lint-staged documentation). Could you either add those to this PR or remove the references to them from this document?

Comment (hoo761) [reply CONTRIBUTING.md:1]: Ok, I removed referances to all tools that are not set up yet. I will add those sections back as they are added.

Comment (saulshanabrook) [reply CONTRIBUTING.md:1]: Thanks!

**Pull request merged**